internal/handlers/updateMetricHandler.go

```diff
@@ -0,0 +1,35 @@
+package handlers
+
+import (
+	"github.com/go-chi/chi/v5"
+	"github.com/sinobite/go-metrics/internal/storage"
+	"github.com/sinobite/go-metrics/internal/utils"
+	"net/http"
+)
+
+func UpdateMetricHandler(writer http.ResponseWriter, request *http.Request) {
+	metricType := chi.URLParam(request, "metricType")
+	metricName := chi.URLParam(request, "metricName")
+	metricValue := chi.URLParam(request, "metricValue")
+
+	if metricType == "gauge" {
+		gaugeValue, err := utils.ConvertToGauge(metricValue)
+		if err != nil {
+			writer.WriteHeader(http.StatusBadRequest)
```

Comment:
Если произошла ошибка, стоит прервать выполнение хендлера:
```suggestion
			http.Error(writer, "Invalid metric value", http.StatusBadRequest)
		         return
```